index.html

```diff
@@ -0,0 +1,3177 @@
+                  <div>
+                    <a href="#" class="d_block">
+                      <span class="sponsor_img si_9 d_block"></span>
+                    </a>
+                  </div>
+                </div>
+                <div class="info">
+                  <h3 class="d_ib">How Can a Good Business Education Benefit You?</h3>
+                  <div class="summary">
+                    Enrolling at a good Business School might be the best investment you can make in
+                    your future, and while costs can be significant, the rewards are incalculable.
+                    Find out more.
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div>
+              <div class="sponsor">
+                <div class="sponsor_wrap pos_rel">
+                  <div class="category d_block pos_abs b_0 l_0 zi_1">
+                    sponsored insight
+                  </div>
+                  <div>
+                    <a href="#" class="d_block">
+                      <span class="sponsor_img si_10 d_block"></span>
+                    </a>
+                  </div>
+                </div>
+                <div class="info">
+                  <h3 class="d_ib">Newsweek Special Edition: Food as Medicine</h3>
+                  <div class="summary">
+                    Humans have been using natural remedies like spices and herbs to heal common
+                    ailments for centuries. Now food science is proving many of those traditions
+                    true and innovating new ways to heal our bodies from the inside out. Discover
+                    the power of plant-based nutrition with this Newsweek Special Edition and start
+                    your journey to a healthier you.
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div>
+              <div class="sponsor">
+                <div class="sponsor_wrap pos_rel">
+                  <div class="category d_block pos_abs b_0 l_0 zi_1">
+                    Rankings
+                  </div>
+                  <div>
+                    <a href="#" class="d_block">
+                      <span class="sponsor_img si_11 d_block"></span>
+                    </a>
+                  </div>
+                </div>
+                <div class="info">
+                  <h3 class="d_ib">Best Infection Prevention Products</h3>
+                  <div class="summary">
+                    During the coronavirus outbreak, health care is more important than ever. In
+                    Newsweek's first Best Health Care series, we highlight the best infection
+                    prevention products.
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="extra">
+        <div class="in_magazine">
+          <div>
+            <div class="magazine">
```

Comment:
One of these should be `section` or `article` element. Let's make the overall page more semantic.


Please do this for all other occurrences

Reply:
Thanks for the review, I'll work on this.